pubspec.yaml

```
@@ -33,6 +33,16 @@ dependencies:
   archive: ^3.6.1
   flutter_svg: ^2.0.14
   image: ^4.3.0
+  flutter_silero_vad:
+    git: 
```

yamllint

[error] trailing spaces

(trailing-spaces)